packages/styles/src/components/form-check.scss

```diff
@@ -48,20 +66,22 @@
   }
 
   &-input {
+    &:not([disabled]),
+    &:not([disabled]) ~ .form-check-label {
+      cursor: pointer;
+    }
+
+    @include utility-mx.focus-style-none();
+
     display: inline-flex;
     flex: 0 auto;
     appearance: none;
     background: transparent;
-    height: form-check.$form-check-input-size;
-    width: form-check.$form-check-input-size;
-    border: form-check.$form-check-input-border-width solid currentColor;
+    height: tokens.get('radio-button-icon-ring');
+    width: tokens.get('radio-button-icon-ring');
+    border: tokens.get('radio-button-icon-border-width') solid
+      tokens.get('radio-button-enabled-stroke');
     margin-top: 1px;
```

Comment:
What is this margin for? Can it be removed? 
If not can you add a comment to explain it?